tests/README.md

```diff
@@ -0,0 +1,8 @@
+To get tests to run, you need to download chromedriver from https://sites.google.com/a/chromium.org/chromedriver/ and put it in `tests/`
+You also need to `pip3 install aloe selenium`
```

Comment:
Not sure if it should or shouldn't be included, but OSX also needed `pip install certifi` first: https://stackoverflow.com/a/42512180

Reply:
Gotcha! I must have had that already, because I didn't need to install a fresh copy for this project. I'll add that to the docs.